test-apps/display-test-app/src/frontend/DisplayTestApp.ts

```diff
@@ -146,6 +146,18 @@ function setConfigurationResults(): [renderSystemOptions: RenderSystem.Options,
   return [renderSystemOptions, tileAdminProps];
 }
 
+// simple function to extract file name, without path or extension, on Windows or Linux
+function getFileName(path: string): string {
```

Comment:
Seems like `path` ought to provide a more straightforward cross-platform way to do this - did you look?

Reply:
Oh, right. Forget I said anything.